ServiceLayer/Services.cs

```diff
@@ -60,12 +60,49 @@ public Result GetBookById(int id)
 
         }
 
-        public bool Update(Book book)
+        public Result Update(Book book)
         {
-            return _BookRepository.ReplaceEmployee(book);
+            Result result;
+            result = Validate(book);
+
+            if (result.ErrorMessage != null)
+                return result;
+            if( _BookRepository.ReplaceEmployee(book))
+            {
+                return result;
+
+            }
+            else
+            {
+                result.ErrorMessage = "No Id Match";
+                return result;
+            }
```

Comment:
Append the error messages with one another so that it will display the message containing all the errors at once

Reply:
Ok